CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 8.7.0
+* Adding support for new ocp_version_ranges by @lipoja in https://github.com/release-engineering/iib/pull/653
```

Comment:
Could you add https://github.com/release-engineering/iib/pull/648 as well? I merged it some time ago.
We should merge couple of more renovate PRs and add it here so they can be released as well

Reply:
Done. Merged a couple more. For others, the CI is failing due to conflicts. Let's hold off on them and new ones to not complicate the release further.

Reply:
I agree